src/libs/API.js

```diff
@@ -1089,6 +1089,14 @@ function Inbox_CallUser(parameters) {
     return Network.post(commandName, parameters);
 }
 
+/**
+ * @returns {Promise}
```

Comment:
nab: not a requirement, but like some of the other functions in this file, it could be nice to add one sentence about what it does in the doc.